package.json

```diff
@@ -9,7 +9,7 @@
     "build": "hexo generate",
     "eslint": "eslint .",
     "format:md": "prettier --write */**/docs/*.md */**/api/*.md",
-    "prepare": "husky"
+    "prepare": "node .husky/install.mjs"
```

Comment:
@uiolee `husky` provides a first-party approach: https://typicode.github.io/husky/how-to.html#ci-server-and-docker

Let's use this instead.